image/cli/mascli/functions/pipeline_config_manage_storage_classes

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+function config_pipeline_manage_storage_classes() {
+  echo ""
+  echo "Trying to auto-detect available storage classes for Manage application..."
+  echo ""
+  reset_colors
+  # 1. ROKS
+  oc get storageclass ibmc-file-gold-gid &>> $LOGFILE
+  if [[ $? == "0" ]]; then
+    MANAGE_STORAGE_CLASS_PROVIDER=ibmc
+    MANAGE_STORAGE_CLASS_RWX=ibmc-file-gold-gid
```

Comment:
Why does manage need it's own RWX/RWO storage class defined?

Reply:
made the required changes to reuse the general RWX or RWO (if it's SNO) storage classes defined.